my-dsl.md

```diff
@@ -3,37 +3,82 @@
 _What is the name of the language? Link the name to its webpage
 (if appropriate)._
 
+[Exhibit](http://www.simile-widgets.org/exhibit3)
```

Comment:
You dont actually use any particular features of markdown in your my-dsl.md file except for your very first response here - it would probably be a good idea to include some!